addons/mass_mailing/static/src/views/mailing_contact_view_kanban.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import { KanbanController } from "@web/views/kanban/kanban_controller";
+import { kanbanView } from '@web/views/kanban/kanban_view';
+import { registry } from '@web/core/registry';
+import { useService } from "@web/core/utils/hooks";
+
+export class MailingContactController extends KanbanController {
```

Comment:
I've noticed than now the handlers like "onImport" seem to be placed on the Renderer in owl.

But it seems to work fine on the Controller too, not sure if it matters.